### ISSUE TITLE
Update the opentelemetry.io registry with Java instrumentations

### ISSUE THREAD

**Issue by trask** [open-telemetry/opentelemetry-java-instrumentation#6385]
And a strategy to keep it up-to-date would be great.

**Comment by trask** [comment]
@chalin @cartermp @austinlparker is there any prior art for keeping the registry up-to-date with instrumentations that we publish? (and should this be a goal?)

**Comment by cartermp** [comment]
Nothing I know of other than people updating entries 😬

**Comment by chalin** [comment]
No, but it is something that we can look into. Can you give one or a few examples of entries that are out-of-date?

**Comment by jaydeluca** [comment]
@svrnm are you using any kind of automated procedure for updates like https://github.com/open-telemetry/opentelemetry.io/pull/5625 or are you doing this work manually?

**Comment by svrnm** [comment]
@jaydeluca there are currently 2 things we do to keep things (semi)automatically updated:

* [registry scanner](https://github.com/open-telemetry/opentelemetry.io/tree/main/scripts/registry-scanner) is a tiny tool that scans most repos (including) java and generates some files from new entries. This is not automatic right now because those files need manual intervention
* https://github.com/open-telemetry/opentelemetry.io/blob/main/.github/workflows/scripts/update-registry-versions.sh#L64 updates the version from packages that have them from the maven registry. It's a fairly new addition, afaik there are currently 2 or 3 java packages with version annotation

I have this outstanding community proposal to rework the registry data: https://github.com/open-telemetry/community/pull/2246

**Comment by jaydeluca** [comment]
Fixed by #14697 (and some other things I have in the works)
